docs/modules/ROOT/pages/how-tos/upgrade-v7-v8.adoc

```diff
@@ -0,0 +1,65 @@
+= Upgrading to Component Version v8.0.0
+
+The component was updated to use `HostNetwork` for the Ceph-CSI driver pods and to disable holder pods.
+
+Using `HostNetwork` for the Ceph-CSI driver pods fixes broken mounts after the Ceph-CSI driver is updated.
+Holder pods will be deprecated in one of the next Rook-Ceph releases.
+
+This how-to is intended for users which are upgrading an existing Rook-Ceph setup to component version v8.0.0 from a previous component version.
+
+== Prerequisites
+
+* `cluster-admin` access to the cluster
+* `kubectl`
+* `jq`
+
+== Steps
+
+. Upgrade the component to version v8.0.0 and wait for the upgrade to finish.
+Ensure the ArgoCD application is synced, in a healthy state, and the OSDs were restarted.
+
+. Delete the Ceph-CSI driver configmap:
++
+[source,bash]
+----
+kubectl --as=cluster-admin delete cm -n syn-rook-ceph-operator rook-ceph-csi-config
+----
+
+. Restart the Rook-Ceph operator:
++
+[source,bash]
+----
+kubectl --as=cluster-admin delete pod -n syn-rook-ceph-operator -l app=rook-ceph-operator
+----
+
+. Delete the Ceph-CSI driver pods:
++
+[source,bash]
+----
+kubectl --as=cluster-admin -n syn-rook-ceph-operator delete pod -l app=csi-cephfsplugin
+kubectl --as=cluster-admin -n syn-rook-ceph-operator delete pod -l app=csi-rbdplugin
+----
+
+. Delete holder pods
+
+[WARNING]
+====
+Running pods will still use the holder pods.
+You should evict the pods by draining each node to remount the volumes.
```

Comment:
Is the idea here to do this anyway combined with a maintenance? Otherwise it might make sense to add an adapted version of https://kb.vshn.ch/oc4/how-tos/force-reboot.html to drain all nodes in a loop with a single command

Reply:
> Is the idea here to do this anyway combined with a maintenance?

That was the idea for us.

Reply:
https://kb.vshn.ch/oc4/how-tos/force-reboot.html was linked in https://github.com/projectsyn/component-rook-ceph/pull/160/commits/3fdb2c1d671ceefc40e8fca7e2fe49a766f0c474